### PR TITLE
feat: add tx internal transfers to activities

### PR DESCRIPTION
**Pull request by sborrazas**
refs #725

### CODE REVIEW

File: test/ae_mdw_web/controllers/activities_controller_test.exs [hunk 1]
```diff
@@ -324,6 +324,87 @@ defmodule AeMdwWeb.ActivitiesControllerTest do
...
+               "type" => "InternalTransferEvent",
+               "payload" => %{
+                 "kind" => "fee_lock_name",
+                 "amount" => 20
+               }
+             } = activity2
+
+      assert %URI{query: query} = URI.parse(next_url)
+
+      assert %{"cursor" => _cursor, "direction" => "forward", "limit" => "2"} =
```
Comment (jyeshe) [inline test/ae_mdw_web/controllers/activities_controller_test.exs:404]: May it make a request to the `next_url` to test the cursor deserialization?

File: test/integration/ae_mdw_web/controllers/activity_controller_test.exs [hunk 2]
```diff
@@ -183,4 +183,26 @@ defmodule Integration.AeMdwWeb.ActivityControllerTest do
              ] = events
     end
   end
+
+  test "it gets tx-based internal transfer events", %{conn: conn} do
```
Comment (jyeshe) [inline test/integration/ae_mdw_web/controllers/activity_controller_test.exs:187]: Could we have an extra one that would have both generational and transactional internal transfer events and test the pagination if it skips both internal transfers on the next page?

**Review by jyeshe — COMMENTED** [review]
Code looks good for me. Only some test automation requests.

**Review by thepiwo — APPROVED** [review]

File: test/ae_mdw_web/controllers/activities_controller_test.exs (hunk 1 above)
Comment (sborrazas) [reply test/ae_mdw_web/controllers/activities_controller_test.exs]: Done

File: test/integration/ae_mdw_web/controllers/activity_controller_test.exs (hunk 2 above)
Comment (sborrazas) [reply test/integration/ae_mdw_web/controllers/activity_controller_test.exs]: I searched the entire transfers database and wasn't able to find a generation that contains both internal and external transfers from the same account

Comment (jyeshe) [reply test/integration/ae_mdw_web/controllers/activity_controller_test.exs]: Thanks, could you add in the CI then? It's possible for an account that receives a dev reward to also create an oracle, right?

Comment (sborrazas) [reply test/integration/ae_mdw_web/controllers/activity_controller_test.exs]: Added a unit tests with both cases combined now, thanks

**Review by jyeshe — APPROVED** [review]

**Pull request merged**